legend-depot-core-services/src/main/java/org/finos/legend/depot/services/projects/DependenciesCache.java

```diff
@@ -52,10 +53,10 @@ private void initCache()
         try
         {
             List<StoreProjectVersionData> allProjectsVersions = projectsVersionsStore.getAll();
```

Comment:
didnt change the creation of map because some dependencies would still be having master and there might be some relying on a project with no dependencies, atleast we would still want to load the cache

Reply:
ok that makes sense but only master snapshot versions can depend on master-snapshot ..maybe we need to do a check for that when calculating deps